examples/MvcWeb/Services/ProjectTypeService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using MvcWeb.TheraLang.Entities;
+using MvcWeb.TheraLang.UnitOfWork;
+
+namespace MvcWeb.Services
+{
+    public class ProjectTypeService : IProjectTypeService
+    {
+        private readonly IUnitOfWork _uow;
+        public ProjectTypeService(IUnitOfWork unitOfWork)
+        {
+            _uow = unitOfWork;
+        }
+        public IEnumerable<ProjectType> GetAllProjectsType()
+        {
+            return _uow.Repository<ProjectType>().Get().AsNoTracking().ToList();
+        }
+
+        public ProjectType GetProjectTypeById(int id)
+        {
+            try
+            {               
+                ProjectType projectType = _uow.Repository<ProjectType>().Get().FirstOrDefault(p => p.Id == id);         
+                return projectType;
+            }
+            catch (Exception ex)
+            {
+                throw new Exception($"Error when geting project by {nameof(id)} = {id}: ", ex);
+            }
+        }
+
+        public async Task Add(ProjectType projectType)
+        {
+                var projecType = new ProjectType { TypeName = projectType.TypeName };               
```

Comment:
```suggestion
                var projectType = new ProjectType { TypeName = projectType.TypeName };               
```

Reply:
can`t name it so, as you have sugested, because in function i take the same argument